GitHub-Org-Management-Policy.md

```diff
@@ -48,22 +40,6 @@ Any repository created under the Node.js GitHub Organization is considered to be
 a project under the ownership of the Node.js Foundation, and thereby subject
 to the Intellectual Property and Governance policies of the Foundation.
 
-No repository may be deleted, transferred in to, or transferred out of, the
-Node.js Foundation GitHub Organization without a simple majority of both
-the TSC and CommComm in *favor* of the action. In certain cases, Node.js
-Foundation Board of Directors approval may also be required.
-
-All repositories must have a README that clearly identifies the purpose of the
-repository, governance documentation that details how the repository is managed,
-and an indication of whether the repository falls under the operational
-direction of either the TSC or CommComm. 
-
-Repositories that are under the operational direction of the TSC are subject to
-TSC oversight.
-
-Repositories that are under the operational direction of the Community Committee
-are subject to CommComm oversight.
```

Comment:
It’s not quite obvious to me why this is being removed … because we don’t currently follow it?

Reply:
Sort of. Also, it's redundant (or maybe just has some nuance I'm missing?). It basically can be paraphrased as: "Repositories that are under CommComm oversight are subject to CommComm oversight." Like, I read that, and other than including CommComm, I'm not sure why it's even there.

Reply:
I agree its bad to have a doc that does not reflect reality but I think instead of removing references to the Community Committee having oversight as the interim solution, we should be making the CommComm chairs owners in addition to all TSE members.  I don't think its our final state but I think its a more acceptable interim state than simply removing any CommComm onership.

Reply:
@mhdawson If you would like to add a commit to make that change, feel free. We will need everyone who has approved to re-approve, I think (or remove their approval if they don't approve). My intention had been to open a PR to that effect immediately after this lands, but if you think it's better to do that now, I'm certainly OK with seeing what everyone thinks about that.